app/pages/[teamSlug]/team/[userId].page.tsx

```diff
@@ -4,14 +4,30 @@ import {
   useProjectMembershipCreateMutation,
   useProjectMembershipDeleteMutation,
   useTeamProjectsQuery,
+  useUserCapacityUpdateMutation,
   useUserQuery,
   useUserRoleUpdateMutation,
 } from '../../../frontend/generated/graphql'
 import Image from 'next/image'
-import { Button } from '../../../frontend/components/button/button'
 import { useRouter } from 'next/router'
 import { ProtectedPage } from '../../../frontend/components/protectedPage'
 import { Toggle } from '../../../frontend/components/toggle/toggle'
+import { useForm } from 'react-hook-form'
+import { InputField } from '../../../frontend/components/inputField/inputField'
+import { Button } from '../../../frontend/components/button/button'
+
+const CAPASITY_MINUTES_FIELD = 'availableMinutesPerWeek'
+
+function validateCapacityField(input: string): string | undefined {
+  const onlyNummber = /^\d+$/
+  const negativeNummber = /^-\d+$/
+
+  if (!input) return "Capacity minutes can't be empty"
+  else if (negativeNummber.test(input)) return "Minutes can't be negative"
+  else if (!onlyNummber.test(input)) return 'Capacity minutes should be number'
```

Comment:
- try to get rid of regex
- allow empty string for setting the capacity to null

Reply:
we will do it later